code/modules/jobs/jobs.dm

```diff
@@ -59,6 +59,21 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 	"Cyborg",
 	"pAI"))
 
+GLOBAL_LIST_INIT(exp_jobsmap, list(

```

Comment:
Also: there may (depending on how much of this you plan to port) later be EXP types that are not based on jobs.